android/fastlane/Fastfile

```diff
@@ -36,15 +41,16 @@ lane :android_build_internal do
 
   versionName = "#{git_commit}-#{git_branch}"
 
-  gradle(task: "clean bundleMosipRelease")
+  gradle(task: "clean bundle#{INJI_FLAVOR}Release")
   upload_to_play_store(
   track: 'internal',
   release_status: 'completed',
   version_name: versionName,
+  package_name: applicationId,
   )
 
   slack(
-      message: "Inji - #{versionName} (#{current_build_number}) is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.", 
+      message: "Inji - #{INJI_FLAVOR} Wallet- #{versionName} (#{current_build_number}) is uploaded to Play store. Description : #{PLAY_CONSOLE_RELEASE_DESCRIPTION}.", 
```

Comment:
Can we use app_name created for each flavor in build gradle

Reply:
Have used a function to get the app name.